tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+   "source": [
+    "## Molecular Dynamics Simulation of Protein\n",
+    "\n",
+    "### Protein System Setup\n",
+    "\n",
+    "In this section, we will obtain our protein coordinates and perform some routine Molecular Dynamics calculations on them.\n",
+    "\n",
+    "For this tutorial, we will use the HIV-1 protease structure (1HSG). It is a homodimer with two chains of 99 residues each. The .pdb file of the protein can be obtained from the protein data bank (https://www.rcsb.org/), but it is probably easier for now to just copy it from the shared directory to your working directory. In fact, most of the files you need can be found in this directory so if you get confused/stuck/lost etc then you can always look here to check you did things correctly.\n",
+    "\n",
+    "`% cp /home/shared/data/1hsg.pdb .` (path to be verified)\n",
+    "\n",
+    "If you look at this file (using the graphical editors vi or nedit for example, you should immediately see that it has two chains; A and B:\n",
+    "\n",
+    "`% vi 1hsg.pdb`\\\n",
+    "or\\\n",
+    "`% nedit 1hsg.pdb`\n",
+    "\n",
+    "Now we need to prepare our protein for simulation. First of all we will extract only the protein coordinates from the pdb file into a new file called `protein.pdb`. To do this enter the following command:\n",
+    "\n",
+    "`% grep ATOM 1hsg.pdb > protein.pdb`\n",
+    "\n",
+    "Use your preferred text editor to open the protein.pdb file and see how it differs from the 1hsg.pdb file.\n",
+    "\n",
+    "We now need to make sure that all the hydrogens are added to our protein. This process will also generate the\n",
+    "parameter/topology file we need.\n",
+    "\n",
+    "`% gmx pdb2gmx -f protein.pdb -ignh -o protein.gro`\n",
+    "\n",
+    "The program should run and present a list of force-fields from which to select. Select the AMBER99SB-ILDN force field which should be option 6 in the list followed by 1 to select the recommended TIP3P water. If all goes well this should generate several files:\n",
+    "1. topol.top \n",
+    "2. topol_Protein_chain_A.itp \n",
+    "3. topol_Protein_chain_B.itp \n",
+    "3. posre_Protein_chain_A.itp\n",
+    "4. posre_Protein_chain_B.itp\n",
+    "5. protein.gro\n",
+    "\n",
+    "Type:\n",
+    "\n",
+    "`% ls`\n",
+    "\n",
+    "to verify that all the above files have been created and are in your directory.\\\n",
+    "Note that the protein has a net charge of +4e. You should see a line that says \"Total charge\n",
+    "in system 4.000 e\".\n",
+    "\n",
+    "Before we can add water we need to define a box in which to put the protein and the water:\n",
+    "\n",
+    "`% gmx editconf -f protein.gro -box 7 7 7 -c -o boxed.gro`\n",
+    "\n",
+    "This puts the protein in the centre of the box that is 7 nm x 7 nm x 7 nm and creates the resulting file boxed.pdb .\\\n",
+    "Next we need to add water to the system. We will also add ions -enough to neutralize the system and to a\n",
```

Comment:
The adding ions here is a bit messy because we do the solvation and then we then again have a blurb about adding ions. Would it make sense to merge both and only mention it after the solvate call?

Reply:
Rephrased it.